From: Neil Horman <nhorman@tuxdriver.com>
Date: Fri, 3 May 2013 19:38:10 +0000
Subject: MAINTAINERS: Fix fcoe mailing list
References: bnc#909501,FATE#317541
Patch-Mainline: v3.10
Git-commit: f4aaea6d5106f6092a80a648b5a12d31b6cb3335

The FCoE mailing list has moved, update it in the MAINTAINERS file

Signed-off-by: Neil Horman <nhorman@tuxdriver.com>
CC: Robert Love <robert.w.love@intel.com>
Signed-off-by: Robert Love <robert.w.love@intel.com>
Acked-by: Hannes Reinecke <hare@suse.de>
---
 MAINTAINERS | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/MAINTAINERS b/MAINTAINERS
index e01ff3e..9ac7fbf 100644
--- a/MAINTAINERS
+++ b/MAINTAINERS
@@ -2560,7 +2560,7 @@ F:	lib/fault-inject.c
 
 FCOE SUBSYSTEM (libfc, libfcoe, fcoe)
 M:	Robert Love <robert.w.love@intel.com>
-L:	devel@open-fcoe.org
+L:	fcoe-devel@open-fcoe.org
 W:	www.Open-FCoE.org
 S:	Supported
 F:	drivers/scsi/libfc/
-- 
1.8.5.2

